docs/cart-and-checkout-blocks/hooks/hook-alternatives.md

```diff
@@ -0,0 +1,456 @@
+| `woocommerce_product_get_image`                       | ❌                | Use `woocommerce_store_api_cart_item_images` ([PR Link](https://github.com/woocommerce/woocommerce/pull/52310) with example.)                                | 
+| `woocommerce_cart_no_shipping_available_html`         | ❌                | This is not editable                                                                                                                                         |
+| `woocommerce_available_payment_gateways`              | 🔶               | Integration with WC Blocks is still required, beyond unsetting gateways, manipulating the payment gateways here may not work in the Cart and Checkout blocks |                                                                                                   
+| `woocommerce_cart_get_total`                          | ✅                |                                                                                                                                                              |
+| `woocommerce_cart_get_fee_tax`                        | ✅                |                                                                                                                                                              |
+| `woocommerce_cart_get_cart_contents_tax`              | ✅                |                                                                                                                                                              |
+| `woocommerce_cart_get_shipping_tax`                   | ✅                |                                                                                                                                                              |
+| `woocommerce_cart_get_shipping_total`                 | ✅                |                                                                                                                                                              |
+
+### Cart _actions_
+
+
+| Old hook                                     | Works in blocks? | Notes                                                                                                                                                                                 |
+|----------------------------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `woocommerce_before_cart`                    | ❌                | No specific equivalent and no Slot/fill for this area. Maybe adding an inner block to the "Cart items" block could work, but it would have to be positioned manually by the merchant. |
```

Comment:
They can also use render hooks for each block to render something before and after using PHP

Reply:
Nice, I'll add that, thanks.